_setting_up/README.md

```diff
@@ -1,184 +1,114 @@
-2: CRAN packages only                       
-3: None                                     
-4: pegboard (0.7.5 -> a32a7836d...) [GitHub]
-5: varnish  (1.0.2 -> 56d594d27...) [GitHub]
-6: curl     (4.3.2 -> 5.2.1       ) [CRAN]  
-7: httr     (1.4.2 -> 1.4.7       ) [CRAN]  
-8: gh       (1.3.0 -> 1.4.1       ) [CRAN]  
-9: covr     (3.5.1 -> 3.6.4       ) [CRAN]  
-
-Enter one or more numbers, or an empty line to skip updates: 1
-
-The downloaded source packages are in
-	‘/tmp/Rtmpn94NAu/downloaded_packages’
-Downloading GitHub repo carpentries/pegboard@HEAD
-These packages have more recent versions available.
-It is recommended to update all of them.
-Which would you like to update?
-
-1: All                                        
-2: CRAN packages only                         
-3: None                                       
-4: tinkr (0.2.0.9000 -> be564d524...) [GitHub]
-
-Enter one or more numbers, or an empty line to skip updates: 1
-
-── R CMD build ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
-✔  checking for file ‘/tmp/Rtmpn94NAu/remotes50ddb31b7ca85/HealthBioscienceIDEAS-sandpaper-fb14e02/DESCRIPTION’ ...
-─  preparing ‘sandpaper’:
-✔  checking DESCRIPTION meta-information ...
-─  checking for LF line-endings in source and make files and shell scripts (420ms)
-─  checking for empty or unneeded directories
-   Omitted ‘LazyData’ from DESCRIPTION
-─  building ‘sandpaper_0.16.4.tar.gz’
+# How to setup local previews of the Workbench website
+
+## Installing requirements
+### Windows/Mac
+1. If R is not already installed, install R using the following: 
+   * [Windows](https://cran.r-project.org/bin/windows/base/) and use the default settings during the installation of **RGUi 4.x.x**
+   * [Mac](https://cran.r-project.org/bin/macosx/)
+1. **Optional, but recommended** Download [RStudio Desktop](https://posit.co/download/rstudio-desktop/)
+1. From RGui Console or RStudio, enter the following command: 
+   ```r
+   options(repos = c(
+   carpentries = "https://carpentries.r-universe.dev/",
+   CRAN = "https://cran.rstudio.com/"
+   ))
+   install.packages(c("remotes"), dep =TRUE)
+   ```
+1. **For Mac only** Before installing Varnish and Sandpaper, there are some 
+additional packages that will need to be installed. The easiest way is via 
+[Homebrew](https://brew.sh/):
+   ```bash
+   brew install freetype fribidi harfbuzz libpng libgit2
+   ```
+1. Enter the following command to install the local version of varnish. This
+version is needed to incorporate the IDEAS identity for the site. 
+   ```r
+   remotes::install_github("HealthBioscienceIDEAS/varnish", dependencies = TRUE)`
+   ```
+1. Install the sandpaper package. 
+   ```r
+   install.packages(c("sandpaper", "pegboard", "tinkr"),
+   repos = c("https://carpentries.r-universe.dev/", getOption("repos")))
```

Comment:
Ahh - I was looking at a different set of [instructions](https://carpentries.github.io/sandpaper-docs/index.html#infrastructure-r-packages) - I wonder if they should harmonize? 
But I agree this looks better.  But we agree that varnish needs to be installed from local fork first before running this, correct?

Reply:
Ooh interesting, but yeah, installing `sandpaper` with dependencies should be enough, I've never explicitly installed `pegboard` or `tinkr`. Carpentries should indeed harmonise their instructions.

> But we agree that varnish needs to be installed from local fork first before running this, correct?
Good point, it might actually be better to put the `varnish` installation step from the fork _after_ the sandpaper installation step, to make sure the latest installed version is the IDEAS-fork one and don't overwrite it with the carpentries-version

Reply:
Yes, correct!

Reply:
OK - in that case I have updated the README to do sandpaper first and IDEAS fork of varnish second.